packages/app-builder-lib/src/platformPackager.ts

```diff
@@ -769,8 +769,7 @@ export function resolveFunction<T>(executor: T | string, name: string): T {
     p = path.resolve(p)
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const m = require(p)
+  const m = await import(p)
```

Comment:
Is this a backward-compatible change that will still allow cjs?

Reply:
Yes it should be.

Reply:
Can you please double-check with a local test project? Can just use a boilerplate repo/project from here: https://www.electron.build/#boilerplates

It's either that route or we need to add explicit `beforeBuild.js` and `beforeBuild.mjs` in the `test/fixtures` + `BuildTest.ts` tests, which is fine, but just much longer of a route.